### ISSUE TITLE
NativeWindowSettings.StartVisible = false does not work on Windows

### ISSUE THREAD

**Issue by seanofw** [opentk/opentk#1197]
### Description

It appears that the `NativeWindowSettings.StartVisible` option doesn't work in OpenTK 4.x on Windows.

### Repro steps

This simple test program wrongly displays a window when run on Microsoft Windows 10 / .NET Core 3.1 / OpenTK 4.2:

```c#
using System;
using OpenTK.Windowing.Common;
using OpenTK.Windowing.Desktop;

public class Program
{
    static void Main()
    {
        using (var window = new GameWindow(GameWindowSettings.Default,
            new NativeWindowSettings { StartVisible = false }))
        {
            window.Run();
        }
    }
}
```

### Expected behavior

The expected behavior should be that no window appears when `StartVisible = false`.

### Actual behavior

The actual result is that a window appears.

### Related information 

* Operating system: Windows 10
* Backend: GLFW 3.3
* Version: OpenTK 4.2, obtained from nuget.org
* .NET Core 3.1
* Workaround: Position the window offscreen initially; assign `IsVisible = false` in the window's constructor; then move it back to the desired location onscreen.

My investigation:

On the C# side, `GameWindow.IsVisible` is `false` after the window is created, even though the window itself is visible!  Reading through the OpenTK source, it _looks_ like OpenTK is invoking `glfwWindowHint()` with the correct values.  Reading through [the GLFW source code](https://github.com/glfw/glfw/blob/master/src/window.c), it _looks_ like `glfwCreateWindow()` is doing the right things with the window hint.  Putting all that together, I'm not sure why this bug is happening, but the `StartVisible` option definitely doesn't work in any of my test cases.  Possibly a P/Invoke issue?

**Comment by seanofw** [comment]
Link to pull request with a fix in it for this issue: https://github.com/opentk/opentk/pull/1199

**Comment by varon** [comment]
Closed by #1199  which is merged!

Thank you again for this fantastic investigation.

**Comment by seanofw** [comment]
Very welcome!  If I find any other bugs, I'll happily submit pull requests to fix them, but other than this issue, OpenTK's been a pretty solid tool so far :)
